src/components/Toggle/Toggle.js

```diff
@@ -18,7 +18,11 @@ const Toggle = ({
   toggled,
   onChange,
   onToggle,
-  id,
+  id = (this.inputId =
```

Comment:
Just noticed this change, I think we could use: https://github.com/IBM/carbon-components-react/blob/master/src/tools/setupGetInstanceId.js for this as well if you want:

```js
import setupGetInstanceId from '../../tools/setupGetInstanceId';

const getInstanceId = setupGetInstanceId();

const Toggle = ({
  id = `__carbon-toggle__${getInstanceId()}`
}) => {
  // ...
};
```

Is the usage of `this` so that it's consistent across re-renders? Was surprised to see `this` in an arrow function which is what lead me to think about other ways to write this 🤔 

One note though, I believe Security folks mentioned that default params in arrow functions is something that's unsupported by Edge (or something quirky like that). Do you remember this coming up? I might have to dig if not 😅

Reply:
@joshblack Oops I totally forgot `<Toggle>` was stateless, have to convert to a class. Did it and made a change to use your `setupGetInstanceId` utility.

Reply:
BTW:

> Is the usage of `this` so that it's consistent across re-renders?

Yes. In other words, to avoid redundant DOM updates upon re-rendering.